syntax = "proto3";

message ProductToSearch{
    int32 id_product = 1;
}

message ProductAvailabilityResponse{
    bool status_code = 1;
}

service ProductAvailability{
    rpc SearchProduct(ProductToSearch) returns (ProductAvailabilityResponse) {}
}